### ISSUE TITLE
Testing needed for the new ways of zscoring

### ISSUE THREAD

**Issue by GWeindel** [GWeindel/hsmm_mvpy#57]
PR #54 introduced a new default way of scoring the data. The previous behavior of the hmp.transform_data() function was to zscore on a by trial basis, the new default way zscore the data by-participant, a third way is to score all the data at once.

It is yet unclear what the best default option is. Further testing is needed to validate the default behavior

**Comment by GWeindel** [comment]
I tested this issue by re-running Figure 3 of the preprint (see caption of the paper).

This is the original result (zscore per participant/all in the case of only one participant per dataset):
![amp-tol-nev](https://github.com/GWeindel/hmp/assets/28750455/9f475591-2b53-4456-957a-d5a5e306cf93)

This is when szcoring each trial:
![amp-tol-nev_pertrial](https://github.com/GWeindel/hmp/assets/28750455/c9cb5b2e-e505-4edf-b6ac-30e657cc6ef6)

This is without zscoring:
![amp-tol-nev_general-szcoring](https://github.com/GWeindel/hmp/assets/28750455/4f007d05-2ec4-49b5-a15b-2a1a66265ee8)

Overall it is clear that zscoring per trial has the best performance at least in the case of one dataset (one participant one condition). It maybe that zscoring this way can change condition differences, this is still to be tested but I suggest that we move to this default as it also improve things on real data

**Comment by jelmerborst** [comment]
I agree. I would also make zscore_acrossPCs the default: https://github.com/GWeindel/hmp/blob/increasing_location/src/hsmm_mvpy/utils.py#L550.

**Comment by GWeindel** [comment]
Right, I'll try that also

**Comment by GWeindel** [comment]
Can I just take that part from your branch so I don't press you too much to be ready for a PR? :)

**Comment by GWeindel** [comment]
It looks like doing this zscoring across PCs is making things worse
![amp-tol-nev_pertrial_acrossPCs](https://github.com/GWeindel/hmp/assets/28750455/4c7ca2a5-5d52-40bb-b0ea-7e2436593fa0)
I can also output the models coefficients but I think graphically it's enough to say that in the case of these simulations it's not a good strategy. I tried on real data and this also seems to worsen things.

My take on this is that it's not a problem if PCs have the same variance as the magnitudes of HMP can pick or not on each PCs (see matrix plot of tutorial 1). However having very low variance components can be problematic in the estimation procedure.
